src/api/emailTemplates.ts

```diff
@@ -32,8 +32,8 @@ const emailTemplates = {
                     "",
                 ],
                 CC: [requestData.supervisor.EMail],
-                Subject: `OSF Action: RPA ${requestData.positionTitle} is pending OSF review.`,
-                Body: `This email has been generated to inform you that a Request for Personnel Action (RPA) for - ${requestData.positionTitle} - has been submitted and is pending the OSF review.
+                Subject: `OSF Action: An incentive package for RPA ${requestData.positionTitle} is ready for OSF review/approval.`,
+                Body: `This email is to inform you that an incentive package Request for Personnel Action (RPA) ${requestData.positionTitle} is pending OSF approval.
```

Comment:
This message should not be changed -- this is the PackageReview stage -- card is for the PackageApproval stage -- right not you will have to make it for the PacakgeApproval stage with PackageApproval sub-stage -- when we add in the new approval substages -- the name will change to OSFApproval or something along those lines for the sub stage name